plugins/CoreHome/vue/src/createAngularJsAdapter.ts

```diff
@@ -253,3 +288,27 @@ export default function createAngularJsAdapter<InjectTypes = []>(options: {
 
   return angularJsAdapter;
 }
+
+export function transformAngularJsBoolAttr(v: unknown): boolean|undefined {
+  if (typeof v === 'undefined') {
+    return undefined;
+  }
+
+  if (v === 'true') {
+    return true;
+  }
+
+  return !!v && v > 0 && v !== '0';
+}
+
+export function transformAngularJsIntAttr(v: string): number {
+  if (typeof v === 'undefined') {
+    return undefined;
+  }
+
+  if (v === null) {
+    return null;
+  }
+
+  return parseInt(v, 10);
+}
```

Comment:
Utility functions to deal w/ `@` angularjs attributes that are meant to be booleans/numbers.